index.d.ts

```diff
@@ -16,14 +16,16 @@ export const defaultOptions: Options<string>;
 
 export type DiversityType = "lowercase" | "uppercase" | "symbol" | "number";
 
+export type StrengthType = 'Too weak' | 'Weak' | 'Medium' | 'Strong';
```

Comment:
Could you change this to use double quotes instead of single quotes, since that seems to be the standard in the rest of the repository?